moveit_simple/test/moveit_simple.cpp

```diff
@@ -50,50 +50,185 @@ TEST(MoveitSimpleTest, reachability)
   ASSERT_FALSE(robot.isReachable("waypoint4"));  //stored in the URDF
 }
 
+
 TEST(MoveitSimpleTest, add_trajectory)
 {
   moveit_simple::Robot robot(ros::NodeHandle(), "robot_description", "manipulator");
   const std::string TRAJECTORY_NAME("traj1");
   const Eigen::Affine3d pose = Eigen::Affine3d::Identity(); 
   ros::Duration(2.0).sleep();  //wait for tf tree to populate
+  const moveit_simple::InterpolationType cart = moveit_simple::CARTESIAN;
```

Comment:
No need to create temporary variables here.  The enums provide more than enough context to understand what they mean.